src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -615,5 +619,51 @@ private static Assembly SgenAssemblyResolver(object source, ResolveEventArgs e)
 
             return null;
         }
+
+        private string[] ParseResponseFile(string[] args)
+        {
+            var parsedArgs = new List<string>();
+            foreach (string arg in args)
+            {
+                if (!arg.StartsWith("@"))
+                {
+                    parsedArgs.Add(arg);
+                }
+                else
+                {
+                    string responseFileName = arg.Substring(1);
+                    foreach (string line in ReadResponseFile(responseFileName))
+                    { 
+                        int i = line.Trim().IndexOf(' ');
```

Comment:
It is better not use empty string as splitter. The file path in References might contain the empty string.

Reply:
@huanwu Strings in the rsp file are gonna look like "--reference abc.dll", indexof(' ') here is to find the first space after "--reference"